trace-telemetry-exporter/src/main.rs

```diff
@@ -0,0 +1,245 @@
+use anyhow::Result;
+use chrono::{DateTime, Utc};
+use clap::Parser;
+use metrics::{counter, gauge};
+use metrics_exporter_prometheus::PrometheusBuilder;
+use rdkafka::{
+    consumer::{stream_consumer::StreamConsumer, CommitMode, Consumer},
+    message::Message,
+};
+use std::{
+    collections::HashMap,
+    net::SocketAddr,
+    sync::{Arc, Mutex},
+    time::Duration,
+};
+use supermusr_streaming_types::dat1_digitizer_analog_trace_v1_generated::{
+    digitizer_analog_trace_message_buffer_has_identifier, root_as_digitizer_analog_trace_message,
+    DigitizerAnalogTraceMessage,
+};
+use tokio::{task, time::sleep};
+use tracing::{debug, trace, warn};
+
+type MessageCounts = Arc<Mutex<HashMap<u8, usize>>>;
```

Comment:
If the `u8` is the digitiser ID then make that explicit by using the type in the `common` crate.